### ISSUE TITLE
[FEATURE] Faster, parallel integration tests for wash

### ISSUE THREAD

**Issue by vados-cosmonic** [wasmCloud/wasmCloud#865]
**Is your feature request related to a problem? Please describe.**
Today, E2E tests are slow in `wash`. 

In the past, we needed to run tests serially because there was no mechanism to prevent `wasmcloud` (i.e. `wasmcloud_host` processes) from clobbering each other using the same configuration files.

These days, we have `wash ctx` which manages contexts, but it seems like there is no way to control the context *per command*. This would make it possible to more properly isolate different processes running. 

Even if the `wash ctx` doesn't work, it *should* be possible to stop the *right* host via the lattice (without resorting to trying to find the PID somehow). 

Even if the lattice message sending method didn't work, there are way to control host creation centrally in the testing code, in order to keep track of the PIDs as they're created and increase control. 

Yet another alternative solution is using a docker container, but this would require building a container with the current release of wash to use (i.e. from a  `build.rs` script?) which would give us the process isolation we'd want for free. We could use [`youki`'s `libcontainer`](https://github.com/containers/youki/blob/main/crates/libcontainer) to do the container spawning.

To summarize, there's definitely *a way* to get parallel tests working with isolated hostsl, this issue represents the work to figure that out and implement it.

**Describe the solution you'd like**

Faster (due to parallelism) integration tests for `wash`

**Describe alternatives you've considered**

N/A

**Additional context**
